### ISSUE TITLE
Can you execute source block in org mode?

### ISSUE THREAD

**Issue by littlehome-eugene** [nnicandro/emacs-jupyter#284]
I'm new to this jupyter notebook stuff.

It is very hard to follow what I can do with this emacs package.

right now, I have an org file,  
after modifying the org file, I export it using `ox-ipynb`
Then I can hit run in the jupyter browser. 

Then, I can play around with jupyter-repl which I created using ` M-x jupyter-server-list-kernels`

Is this intended use of this library?

Is there a way for me to directly play around inside the org file without exporting it to ipynb?

**Comment by dcherian** [comment]
this package provides a "backend" for org-babel. See here for some intro docs: https://orgmode.org/worg/org-contrib/babel/

**Comment by dcherian** [comment]
(sorry hit reply too early). Yes, you can execute source blocks (`#+begin_src jupyter-python`) usually with `C-c C-c` with point in a source block.

**Comment by littlehome-eugene** [comment]
Thanks for the input, 

Yes I can see that I can execute the code block in C-c C-c. 
It prompts me with `Evaluate this jupyter-python code block on your system?`

It sounds like it's running code in my local machine.  
What if I want to work on jupyter notebook server(if that makes sense)  

I have tensorflow-based image docker running jupyter and I want to use the library installed in the docker.. 

Oh I could log in myself (with `docker exec -it`) to the docker container and run emacs there inside the container.  
I was thinking running emacs on my host machine though,

**Comment by dcherian** [comment]
Yes that is possible. See https://github.com/nnicandro/emacs-jupyter#communicating-with-kernel-notebook-servers

**Comment by littlehome-eugene** [comment]
Thanks again for help

although it's hard to decrypt what I need to do

> If :session is a TRAMP file name like /jpy:localhost#8888:NAME it is interpreted as corresponding to a connection to a kernel through a Jupyter notebook server located at http://localhost:8888.

I interpreted it as I need to replace `:session` with a file-like thing.. So the below is What I came up with.. but not working.. 

```
#+BEGIN_SRC jupyter-python /jpy:example.com#10008:048010fa-cacc-4371-bff5-4c6cdf617dd6 py
print('hello')
#+END_SRC
```

**Comment by dcherian** [comment]
I do `/jpy:localhost#8888:emacs` and somehow make jupyter set the kernels name to be "emacs". There's a tunnel open on 8888 to a remote jupyter server
